WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldEditorView.swift

```diff
@@ -9,8 +9,10 @@ struct CustomFieldEditorView: View {
 
     private let initialKey: String
     private let initialValue: String
+    private let isCreatingNewField: Bool
```

Comment:
I thought about just checking if `initialKey` is empty instead of adding a new variable here, but at the end I decide to add the variable to make the logic more clear, and also to avoid coupling this logic to the point that empty keys are not allowed.

Reply:
Unless we need this field for any other case, how about keeping `onDelete` optional and render the delete button only if the closure is available?

Reply:
This could work too, I thought about it first but thought maybe it's not clear enough, but now I think it would be simpler, I'll update the code to use it.

Reply:
Done in 9b95760